src/functions/genfrac.js

```diff
@@ -197,7 +209,10 @@ defineFunction({
                 group.value.leftDelim, delimSize, true,
                 options.havingStyle(style), group.mode, ["mopen"]);
         }
-        if (group.value.rightDelim == null) {
+
+        if (group.value.continued) {
+            rightDelim = buildCommon.makeSpan([]); // zero width for \cfrac
```

Comment:
I thought this might cause `\left`/`\right` delimiters to not render, but it doesn't.  I guess the left/right delimeters for `genfrac` are for `a \choose b`.  Do we need a similar `if (group.value.continued)` for `leftDim` up above?

Reply:
I don't think so. In LaTeX, they write a `\kern-\nulldelimiterspace`, but only on the right side of the fraction, not on the left side. We get the same behavior by omitting the space taken by our `\nulldelimiter`.